app/controllers/api/v1/topics_controller.rb

```diff
@@ -0,0 +1,14 @@
+module Api
+  module V1
+    class TopicsController < ApplicationController
+      include Concerns::ActAsApiRequest
+      include DeviseTokenAuth::Concerns::SetUserByToken
```

Comment:
As an enhancement, you can create an api_controller.rb in order to include all common logic for controllers such as the includes in lines 4 and 5. Then you can extend it to all your business logic controllers instead of ApplicationController.
With this, you won't need to include the same concerns, helpers, etc on each controller. Also, it helps to maintain consistency between all controllers.

Reply:
Excellent!